backends/azure_table.go

```diff
@@ -91,7 +91,7 @@ func formattedRequestTime() string {
 func (c *AzureTableBackend) Send(ctx context.Context, req *fasthttp.Request, resp *fasthttp.Response, resourceType string, resourceId string) error {
```

Comment:
Please rename this file and class names from AzureTable to AzureCosmosDB, as this is really an Azure Cosmos DB database backed by a default SQL store.

Reply:
agree. Renamed